cvat-ui/src/components/quality-control/task-quality/jobs-filter-configuration.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { Config } from '@react-awesome-query-builder/antd';
+
+export const config: Partial<Config> = {
+    fields: {
+        state: {
+            label: 'State',
+            type: 'select',
+            operators: ['select_any_in', 'select_equals'],
+            valueSources: ['value'],
+            fieldSettings: {
+                listValues: [
+                    { value: 'new', title: 'new' },
+                    { value: 'in progress', title: 'in progress' },
+                    { value: 'rejected', title: 'rejected' },
+                    { value: 'completed', title: 'completed' },
+                ],
+            },
+        },
+        stage: {
+            label: 'Stage',
+            type: 'select',
+            operators: ['select_any_in', 'select_equals'],
+            valueSources: ['value'],
+            fieldSettings: {
+                listValues: [
+                    { value: 'annotation', title: 'annotation' },
+                    { value: 'validation', title: 'validation' },
+                    { value: 'acceptance', title: 'acceptance' },
+                ],
+            },
+        },
+        id: {
+            label: 'ID',
+            type: 'number',
+            operators: ['equal', 'between', 'greater', 'greater_or_equal', 'less', 'less_or_equal'],
+            fieldSettings: { min: 0 },
+            valueSources: ['value'],
+        },
+        task_id: {
+            label: 'Task ID',
+            type: 'number',
+            operators: ['equal', 'between', 'greater', 'greater_or_equal', 'less', 'less_or_equal'],
+            fieldSettings: { min: 0 },
+            valueSources: ['value'],
+        },
+        task_name: {
+            label: 'Task name',
+            type: 'text',
+            valueSources: ['value'],
+            operators: ['like'],
+        },
+        project_name: {
+            label: 'Project name',
+            type: 'text',
+            valueSources: ['value'],
+            operators: ['like'],
+        },
```

Comment:
As soon as we can launch quality checks no more than in just 1 project, project-level conditions seem to be extra.

Reply:
Removed